roles/setup_etcd/README.md

```diff
@@ -0,0 +1,148 @@
+# Setup ETCD
+
+This Ansible Galaxy Role installs and configure ETCD for Patroni cluster setup.
+
+Note: The role by default installs 3.5.6
+
+Not all Distribution or versions are supported on all the operating systems available.
+
+Note: The role does not configure EDB Postgres Advanced Server or PostgreSQL for replication it only installs ETCD across multiple nodes.
+
+The ansible playbook must be executed under an account that has full privileges.
+
+## Requirements
+
+The requirements for this ansible galaxy role are:
+
+  1. Ansible >= 2.9
+  2. `community.general`
+  3. `edb_devops.edb_postgres` -> `setup_repo` - for installing the EPAS/PG
+     repository
+  4. `edb_devops.edb_postgres` -> `install_dbserver` - for installing the EPAS/PG
```

Comment:
I don't think `install_dbserver` is required for setting up etcd

Reply:
fixed